setup.py

```diff
@@ -578,14 +587,12 @@ def pxd(name):
                      '_libs/tslibs/frequencies']},
     '_libs.tslibs.parsing': {
         'pyxfile': '_libs/tslibs/parsing',
-        'pxdfiles': ['_libs/src/util',
```

Comment:
side note, should parsing depend on nattype? (then for example nat strings are in 1 place).

Reply:
Well nat_strings is defined but not c-defined in nattype, so we could import it into parsing without needing to specify the dependency.

I'd be fine with that, or with having it defined one extra place.  parsing has no intra-pandas dependencies at the moment (neither import nor cimport) which makes it really low-stress.